src/bento/userDetailViewData.js

```diff
@@ -0,0 +1,33 @@
+import { EDIT } from './adminData';
+
+export const getColumnInfo = (removeRenderer, approvedRenderer, accessType) => [
+  { name: 'arm', label: 'Arms' },
+  { name: 'date', label: 'Request Date' },
+  { name: 'date1', label: 'Approved Date' },
+  {
+    name: 'date2',
+    label: 'Approved By',
+    options: {
+      customBodyRender: approvedRenderer,
+    },
+  },
+  {
+    name: 'remove',
+    label: 'Remove',
+    options: {
+      display: (accessType === EDIT),
```

Comment:
@ogwuazors Same requirement like above. We need to make custodian configuration file simple. Can we move this comparison in view? If not, we can discuss other options as well.

Reply:
working on this.